scripts/tuple_tree_generator/tuple_tree_generator/definition/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .definition import Definition
```

Comment:
Please put license notice *everywhere*.

```
#
# This file is distributed under the MIT License. See LICENSE.md for details.
#
```

Reply:
We really need a check for the license notice in `check-revng-conventions`

Reply:
Makes sense, but maybe not now.
Feel free to open an issue for this.